### PR TITLE
Home about pages

### PR DESCRIPTION
**Pull request by Diana-Duya**
In this PR:

- I created a new repository and cloned it to my local computer.
- Next I set up the linters and created the HTML, CSS and JavaScript files
- I also added the necessary images and icons to the images folder.
- Next, I modified the html, CSS and JavaScript files to create, style the website and also to add features like the mobile-menu.
- The speaker section is  created dynamically in the script.js file.
- I committed the changes after running and fixing linter errors.

Here is a link to the [loom video](https://www.loom.com/share/fa8de27d51b7455097af82a0f88ef136) where i explain more about this project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                <span class="bar bar2"></span>
+                <span class="bar bar3"></span>
+            </div>
+            <div class="main-menu">               
+                    <a class="nav-item nav-active" href="about.html">About</a>
+                     <a class="nav-item" href="">Program</a>
+                    <a  class="nav-item"  href="">Join</a>
+                    <a  class="nav-item" href="">Sponsor</a>
+                   <a  class="nav-item nav-items" href="">Sign in</a>
+            </div>
```
Comment (m05e5) [inline index.html:35]: - [ ] Hi. Great job, but you are required to have the logo of your app on the left-hand side of your nav. Kindly add it following the design provided
image

File: CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,574 @@
...
+  display: block;
+  margin-right: 100%;
+  padding-left: 20px;
+}
+
+.main-menu {
+  position: absolute;
+  width: 96%;
+  height: 500px;
+  display: flex;
```
Comment (m05e5) [inline CSS/style.css:67]: - [ ] Please kindly readjust the width of your nav links. on the provided design the nav links take only the right part of the white nav bar

File: CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,574 @@
...
+
+.list li {
+  align-items: center;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'lato', sans-serif;
```
Comment (m05e5) [inline CSS/style.css:372]: - [ ] Hi you are required to stick to the design as much as possible so kindly give your footer a `flex-direction: row;` with some space between the logo and the text as it is on the provided design

**your design**
image

**DTF(Design to follow)**
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+     </section>
+     <footer class="footer">
+     <div class="end">
+        <img class= end-img src="./img/GDG.png" alt="">
+        <h3 class="camp">Google Dev Group.org</h3>
+     </div>
+     <p class="copyright">©2022</p>
+     </footer>
+    </div>
+</section>
```
Comment (m05e5) [inline about.html]: - [ ] Please you are required to have only the logo and a small note in the about page footer, not partners please kindly fix it 

image

File: README.md [hunk 5]
```diff
@@ -1,72 +1,60 @@
...
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Diana-Duya/Capstone-Project/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
```
Comment (m05e5) [inline README.md:54]: - [ ] Hi you are required to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in your Acknowledgments section, as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Diana-Duya , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders



## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] _Please check the comments under this review_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Diana-Duya** [comment]
Hello @m05e5 , 
Thank you for the review. I have made all the required changes.
- Added a logo
- Added the necessary acknowledgements
- Made adjustments to the footer

**Review by mgmediaweb — APPROVED** [review]
Hi @Diana-Duya, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved6

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**